gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/ResourcePluginCrudServiceInMemory.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package inmemory;
+
+import io.gravitee.apim.core.plugin.crud_service.ResourcePluginCrudService;
+import io.gravitee.apim.core.plugin.model.ResourcePlugin;
+import io.gravitee.rest.api.service.exceptions.ResourceNotFoundException;
+import io.gravitee.rest.api.service.exceptions.TechnicalManagementException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ResourcePluginCrudServiceInMemory implements ResourcePluginCrudService, InMemoryAlternative<ResourcePlugin> {
+
+    final List<ResourcePlugin> storage = new ArrayList<>();
+
+    @Override
+    public ResourcePlugin findById(String resourceId) {
+        if (resourceId == null) {
+            throw new TechnicalManagementException("resourceId should not be null");
```

Comment:
```suggestion
            throw new TechnicalManagementException(resourceId + " should not be null");
```

Reply:
hum nope because this is the case when resourceId is null